CodeEdit/Features/CEWorkspace/Views/CETaskFormView.swift

```diff
@@ -0,0 +1,79 @@
+//
+//  CETaskFormView.swift
+//  CodeEdit
+//
+//  Created by Axel Martinez on 12/4/24.
+//
+
+import SwiftUI
+
+struct CETaskFormView: View {
+    @Binding var task: CETask
+
+    @State private var selectedItemId: UUID?
```

Comment:
I prefer the style
```suggestion
    @State private var selectedItemId: UUID?
    @Binding var task: CETask
```
However, there is no right or wrong here...

Reply:
I don't see a consistent approach on this on the app, so I'll leave it as is to avoid having too many changes and until a style is decided and enforced by SwiftLint.

Reply:
Sorry for the confusion, I was wrong. The correct order is:
1. Environment
2. Bindings
3. State